CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### citus-docker v12.1.4.docker (June 06,2024) ###
+
+* Bump Citus version to 12.1.4
```

Comment:
Why did we stop adding PG version bumps in here?

Should we document it here that we bumped PG versions for 14,15 and 16 ?

Reply:
Added